platform/ios/src/MGLMapboxEvents.m

```diff
@@ -12,8 +12,8 @@
 
 #include <sys/sysctl.h>
 
-static const NSUInteger version = 1;
```

Comment:
@1ec5 And [you've pondered on this before](https://github.com/mapbox/mapbox-gl-native/pull/1642#discussion_r31078613), I see.